ee/end-to-end-install.md

```diff
@@ -39,8 +38,7 @@ docker container run --rm -it --name ucp \
   --interactive
 ```
 
-This runs the install command in interactive mode, so that you're prompted
-for any necessary configuration values.
+This runs the [docker/ucp install](https://docs.docker.com/reference/ucp/3.2/cli/install/) command in interactive mode so that you're prompted for any necessary configuration values.
```

Comment:
should probably be without the domain name, so that it works correctly when (pre)viewing on other domains

```suggestion
This runs the [docker/ucp install](/reference/ucp/3.2/cli/install/) command in interactive mode so that you're prompted for any necessary configuration values.
```